test/quantization/test_quantize.py

```diff
@@ -856,6 +856,50 @@ def test_default_quantized_lstm(self):
 
                 y, (h, c) = cell_dq(x, (h, c))
 
+    def test_per_channel_lstm_quantize(self):
```

Comment:
Once PR 38851 lands, this can be added to an existing test, by looping over an additional qconfig